Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Test/TestIPAddressAC.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+using System;
+
+namespace Promact.Trappist.DomainModel.ApplicationClasses.Test
+{
+    public class TestIPAddressAC
+    {
+        public int Id { get; set; }
+
+        public string IPAddress { get; set; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename property 'IPAddress' to match camel case naming rules, consider using 'IpAddress'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)